koiki/email.py

```diff
@@ -0,0 +1,56 @@
+import os.path
+from django.core.mail import EmailMessage
+from django.template.loader import render_to_string
+from django.utils.translation import gettext as _
+
+from koiki import wcfmmp_host, logger, error_mail_recipients
+
+
+class SuccessDeliveryMail:
+    def __init__(self, delivery):
+        self.pdf_path = delivery.print_pdf()
```

Comment:
We should rather pass the pdf path as an argument. By reducing the coupling this way, we'll make it easier to refactor `print_pdf()` later on. We'll be able to change things there without having to touch this class at all. This will make this class' tests simpler :ok_hand: